README.md

```diff
@@ -150,6 +150,17 @@ void setOptions(int keepAlive, bool cleanSession, int timeout);
 
 Connect to broker using the supplied client id and an optional username and password:
 
+Register a custom clock source ("millis source") callback:
+```c++
+void setClockSource(MQTTClientClockSourceCb);
+// Callback signature: uint32_t customMillis (void) {}
+
+void clearClockSource();
```

Comment:
I would remove `clearClockSource` and make `setClockSource` accept `NULL` to reset to the default implementation.

Reply:
I think passing `NULL` or `nullptr` already works. We just need to add it to the docs.

Reply:
Yes, your are right. It would be easy to pass nullptr and keep code cleaner.